### PR TITLE
 Portfolio with Flex-Cards

### PR DESCRIPTION
**Pull request by ridaarif98**
In this Portfolio,
I add a grid in section
Inside grid flex-box created
Add Images to flexbox
Add a section for 'Myself intro'
Integrate linters 
Replace Span with ul li
Resolve linters error

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -45,6 +45,178 @@ <h1 class="intro">I'm Rida Glad to see you!</h1>
       </ul>
     </div>
   </section>
+
+  <section class="Works">
+
+      <div class="grid-item">   
+        <div class="flexBox1">
+          <img src="C:/git/MyPortfolio/images/Portfolio.svg" alt="MyWork">
```
Comment (Dipeshtwis) [inline Index.html:53]: - [ ] The image is not showing in your design.

File: Index.html [hunk 2]
```diff
@@ -45,6 +45,178 @@ <h1 class="intro">I'm Rida Glad to see you!</h1>
...
+            <li class="year">2015</li>
+          </ul>
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="tags">
+          <span class="html">Html</span> 
+          <span class="css">css</span>
+          <span class="java">javascript</span>
+        </div>
```
Comment (Dipeshtwis) [inline Index.html:97]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

File: Index.html [hunk 3]
```diff
@@ -45,6 +45,178 @@ <h1 class="intro">I'm Rida Glad to see you!</h1>
...
+        <span class="langTags">Javascript</span>
+     </div> 
+     <div class="boxDiv">
+      <img src="C:/git/MyPortfolio/images/Html.svg" alt="JavascriptImg"> 
+       <span class="langTags">Html</span>
+    </div> 
+    <div class="boxDiv">
+      <img src="C:/git/MyPortfolio/images/css.svg" alt="JavascriptImg"> 
+       <span class="langTags">CSS</span>
+    </div>
```
Comment (Dipeshtwis) [inline Index.html:204]: - [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Ismailco , 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of Grid and Flexbox.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- For now, your text is overflowing, Below is the screenshot

Screenshot (1372)

**_Check the comments under the below_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html [hunk 4]
```diff
@@ -45,6 +45,181 @@ <h1 class="intro">I'm Rida Glad to see you!</h1>
...
+        </div>
+       
+        <button class="See-Project">See Project</button>
+
+        </div>
+      </div>
+
+      <div class="grid-item"> 
+        <div class="flexBox4">
+          <img src="C:/git/MyPortfolio/images/FourthPortfolio.svg" alt="MyWork">
```
Comment (henatan99) [inline Index.html:139]: - [ ] You images are not displaying. Please use relative path for image `src`. You can make it like `src="./images/FourthPortfolio.svg"`. Kindly make the sma changes for similar image paths too.

File: Index.html (hunk 1 above)
Comment (henatan99) [reply Index.html]: - [ ] This issue is not addressed! See the inline comment for this below the new review summary! :pray:

File: Index.html (hunk 2 above)
Comment (henatan99) [reply Index.html]: This issue is addressed. Nice work! :clap:

File: Index.html (hunk 3 above)
Comment (henatan99) [reply Index.html]: This issue is addressed. Nice work! :clap:

File: Index.html [hunk 5]
```diff
@@ -45,6 +45,181 @@ <h1 class="intro">I'm Rida Glad to see you!</h1>
       </ul>
     </div>
   </section>
+
+  <section class="Works">
+
+      <div class="grid-item">
```
Comment (henatan99) [inline Index.html:51]: - [ ] The `grid-item` elements have some design issues: alignment, spacing, and typography as already pointed out by the previous reviewer. For reference see the attached screenshot images and compare your clone with the original design. 
Screenshot (4)  
Screenshot (5)
- [ ] OPTIONAL: Semantically, the `grid-item` elements should better be `li` elements and wrapped altogether in a `ul` tag.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi Rida, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 5 above)
Comment (DeliceLydia) [reply Index.html:51]: Well addressed this✔️

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved🎉 🎉 

Hi @ridaarif98, 

Your project is complete! I can see the hard work from your project you nailed it! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Keep Aiming High!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**